### DISCUSSION TITLE
Mod balancing

### DISCUSSION THREAD

**Discussion by NerjalNosk** [RecraftorsMC/Enchantment-Decay#1] [General]
The mod, as it stands, is highly unbalanced.

Hence, any input on how to improve this would be very welcome.

Thank you.

**Comment by KateHanami** [comment]
suggestions: 
buff: add a passive unbreaking / mending effect to enchantment
buff: enchanted weapon / tools will (effectively) never break
buff: broken weapon / tools gets a slight repair when enchantment decays by a level (configurable)
nerf: decay faster the higher level of the enchantment (configurable)
nerf: cannot repair a tool that have multiple enchantments of level III or 1 enchantment of level V (configurable)
nerf: repairing a tool that have less than 50% enchantment left, have a chance to decay (configurable)
balance: use player levels and / or items like bottle of enchanting or lapis lazuli to mend an enchantment (before it fully decays)

**Comment by ManicStarGirl** [comment]
I love this mod in combo with "Limitless Leveling" by lucalabs-de (at least on modrinth) and "Enchantment Transfer" by treekicker4 (again on modrinth). This gives enchanting a level of balance where you can enchant up to any level, but it takes many resources (exps and books) to get to Efficiency 10 or 20, but you lose that instant mining quick, needing more resources to get back to the instant mining. Enchantment Transfer just allows that grind to get better tools faster, as the degrading is very fast.

**Comment by error404unidentified101-droid** [comment]
When the next update?

**Reply by NerjalNosk** [reply]
Difficult to say. I have other more meaningful life matters and projects on the forefront, and while I do intend to update that mod to more modern game versions, it is a bit on the back-burner for me. Hopefully, I would say, before the end of the year I would like to have most of my older mods updated, but there is one priority that is - and has long been - holding me back with some technical difficulties.

Regardless, thank you for the interest you have for that one small mod, and hopefully I can live up to yall's expectations whenever I manage to update it.

**Reply by error404unidentified101-droid** [reply]
That’s fair. It's good to hear you’re prioritizing your personal projects. Technical hurdles are always a challenge, so don’t feel rushed on our account. We appreciate the update and will be look forward to whenever you’re ready to release it.
